src/classes/DragNDropController_Test.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>36.0</apiVersion>
```

Comment:
Please be consistent across the project and keeep the api version up to date (44.0).